src/temperature_component.cpp

```diff
@@ -224,54 +224,66 @@ void TemperatureComponent::prepareToRun() {
         C[i] = 0.0;
     }
 
-    // DOECLIM parameters calculated from constants set in header
+    // Dependent DOECLIM model parameters, based on constants set in the header
+    // Constants & conversion factors
+    kcon = 3155.0;               // conversion factor from cm2/s to m2/yr; TODO do we want to use secs_per_Year / 10000 to be consistent?
```

Comment:
@bpbond part of me wants to change this to `secs_per_Year / 10000` so that it is consistent with the seconds per year but this does end up changing the output, here is the MSE between the old and new output per variable. It seems like a trivial change in output to me, @bpbond what do you think? 

```
# A tibble: 5 x 2
  variable      MSE
  <chr>       <dbl>
1 Ca       1.38e- 6
2 FCO2     1.60e-10
3 Ftot     1.60e-10
4 heatflux 6.28e- 9
5 Tgav     2.73e- 9
```

Reply:
What's MSE here? Yes, looks trivial, and I agree consistency would be nice.

Reply:
mean squared error

Reply:
Definitely trivial!